packages/remix-dev/cli/commands.ts

```diff
@@ -144,6 +144,7 @@ export async function build(
 
   let start = Date.now();
   let config = await readConfig(remixRoot);
+  fse.emptyDirSync(config.assetsBuildDirectory);
```

Comment:
Do we need to do the same thing for `watch` command too?

Reply:
I left out `watch` since the issue at hand here is related for not deploying stale assets on new builds.  I'm unsure if folks run into the same issue when running watch.  Assuming the watcher only does incremental re-builds, I'm not sure we could safely delete everything?